src/validate.rs

```diff
@@ -232,39 +232,14 @@ fn validate_team_members(data: &Data, errors: &mut Vec<String>) {
     });
 }
 
-/// Ensure alumni are not active
+/// Alumni team must consist only of automatically populated alumni from the other teams
 fn validate_alumni(data: &Data, errors: &mut Vec<String>) {
-    let active_members = match data.active_members() {
-        Ok(ms) => ms,
-        Err(e) => {
-            errors.push(e.to_string());
-            return;
-        }
+    let Some(alumni_team) = data.team("alumni") else {
+        return;
```

Comment:
I wonder if we should error here since if the alumni team is removed, presumably this whole check should be removed?

Reply:
Good call. Fixed.